.github/workflows/advertisements-versions.yml

```diff
@@ -0,0 +1,71 @@
+name: Ads Service (Fixed) Test and Build
+
+on:
+  push:
+    branches: [ main ]
+    paths:
+      - ads-service-versions/**
+  workflow_dispatch:
+    branches: [ main ]
+
+defaults:
+  run:
+    working-directory: ads-service-versions
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: us-east-1
+
+    - name: Login to ECR
+      id: login-ecr
+      uses: docker/login-action@v1
+      with:
+        registry: public.ecr.aws
+        username: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        password: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+
+    - name: Build and push
+      uses: docker/build-push-action@v2
+      with:
+        context: ./ads-service-versions
+        file: ./ads-service-versions/ads_v1/Dockerfile
+        platforms: linux/amd64
+        push: true
+        tags: ${{ secrets.PUBLIC_ECR_REGISTRY }}/ddtraining/ads_versions_v1:latest
```

Comment:
- the `ddtraining/ads_versions` repo will need to be created in cloudops first
- rather than having two separate image repos `ddtraining/ads_versions_v1` and `ddtraining/ads_versions_v2`, can we create a single `ddtraining/ads_versions` and update the tag to be `v1` or `v2` accordingly?

Reply:
> * the `ddtraining/ads_versions` repo will need to be created in cloudops first

How can I do that?

> 
>     * rather than having two separate image repos `ddtraining/ads_versions_v1` and `ddtraining/ads_versions_v2`, can we create a single `ddtraining/ads_versions` and update the tag to be `v1` or `v2` accordingly?

The reason why I didn't do that is because I assumed you were using tagging to control the ecommerce workshop releases. I know that using `v1`,`v2` and `v2_1` as part of the repo is not the canonical way to do it, but that way, if we tag all repos as part of a release, we can include these three as well.

Reply:
> > * the `ddtraining/ads_versions` repo will need to be created in cloudops first
> 
> How can I do that?
> 
You can make a PR updating this file with the new repo name https://github.com/DataDog/cloudops/blob/master/AWS/marketing/ecr-public/data.yaml
> > ```
> > * rather than having two separate image repos `ddtraining/ads_versions_v1` and `ddtraining/ads_versions_v2`, can we create a single `ddtraining/ads_versions` and update the tag to be `v1` or `v2` accordingly?
> > ```
> 
> The reason why I didn't do that is because I assumed you were using tagging to control the ecommerce workshop releases. I know that using `v1`,`v2` and `v2_1` as part of the repo is not the canonical way to do it, but that way, if we tag all repos as part of a release, we can include these three as well.

We could do the release tag like `ddtraining/ads_versions:v1-2.2.3`